src/views/tutorial-library/components/current-filters/index.tsx

```diff
@@ -62,13 +62,11 @@ export function CurrentFilters() {
 
 							return (
 								<li key={labelText}>
-									<button
-										onClick={() => item.refine(refinement)}
-										className={currentFiltersStyle.filterButton}
-										aria-label={`Remove filter ${labelText}`}
-									>
-										{labelText} <IconX16 />
-									</button>
+									<Tag
+										buttonAriaLabel={`Remove filter ${labelText}`}
```

Comment:
Can you add a TODO above this to test with voice control as a breadcrumb for later? This gets into [Success Criterion 2.5.3 Label in Name](https://www.w3.org/TR/WCAG21/#label-in-name) territory.

Reply:
Sure thing 👍